### PR TITLE
Login without persona

### PR DESCRIPTION
**Pull request by JanikOskar**
@laniszew @lmazurowski @pumanitro

### CODE REVIEW

File: src/components/CreateSpotAndPersona/CreateCard/EntityCard.tsx [hunk 1]
```diff
@@ -30,11 +33,12 @@ const initialState: ImageRef = {
   blob: null,
 };
 
-export const CreateEntityCard: FC<PropsCard> = ({ initialValues, updateCard, nextPathName }) => {
+export const EntityCard = ({ nextPathName, stepperNumbers, currentNumber, updateCard, initialValues }: PropsCard) => {
```
Comment (laniszew) [inline src/components/CreateSpotAndPersona/CreateCard/EntityCard.tsx]: why not `FC<PropsCard>` ?

File: src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -113,13 +114,19 @@ export const Login: FunctionComponent = () => {
...
+    } else if (token) {
       localStorage.setItem(PS_TOKEN_NAME, token);
       setUser(data?.data?.loginUser?.user || null);
-      history.push('./createpersona');
+      history.push('./scanner');
     }
   };
-
+  {
+    console.log(userPersona);
```
Comment (laniszew) [inline src/pages/Login/Login.tsx]: remove console.log

File: src/components/LoginBar/LoginBar.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  :hover {
+    background-color: rgba(85, 133, 255, 0.2);
+  }
+`;
+
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.utils.background.light};
+  text-decoration: none;
+`;
+export const LoginBar = ({ isLogged }: LoginBarProps) => {
```
Comment (lmazurowski) [inline src/components/LoginBar/LoginBar.tsx]: one line space

File: src/components/LoginBar/LoginBar.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <>
```
Comment (lmazurowski) [inline src/components/LoginBar/LoginBar.tsx:33]: hey - did You update this PR with latest changes from master - I wrote comment about this piece of code in previously created PR?

File: src/components/MyPersona/MyPersona.tsx [hunk 5]
```diff
@@ -145,7 +145,7 @@ export const MyPersona: FC = () => {
           </TextInShare>
         </ShareQr>
       </PageWrapperSpaceBetween>
-      <NavLink to="./createpersona">
+      <NavLink to="./creation/step/1/entity/persona">
```
Comment (lmazurowski) [inline src/components/MyPersona/MyPersona.tsx]: please use APP_ROUTES constant from routes.ts here, or create one and use id here if there is no link already prepared

File: src/components/TermandPrivacy/InformativeClausule.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { RegulationsCards } from './RegulationsCards';
+
+export const InformativeClause = () => {
```
Comment (lmazurowski) [inline src/components/TermandPrivacy/InformativeClausule.tsx]: Why didn't You use an simplefied version? () => (<div/>) ?
And where is FC type for this functional component?

File: src/components/TermandPrivacy/PrivacyAndCookies.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { RegulationsCards } from './RegulationsCards';
+
+export const PrivacyAndCookies = () => {
```
Comment (lmazurowski) [inline src/components/TermandPrivacy/PrivacyAndCookies.tsx]: Why didn't You use an simplified version? () => (

) ?
And where is FC type for this functional component?

File: src/components/TermandPrivacy/RegulationsCards.tsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+const TextInRegulation = styled.div`
+  white-space: pre-wrap;
+  margin-bottom: 8px;
+`;
+
+const NumberArticle = styled.div`
+  margin-bottom: 4px;
+`;
+
+export const RegulationsCards: FC<RegulationsProps> = ({
```
Comment (lmazurowski) [inline src/components/TermandPrivacy/RegulationsCards.tsx:40]: Why didn't You use an simplified version? () => (

) ?

File: src/components/TermandPrivacy/TermOfUse.tsx [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { RegulationsCards } from './RegulationsCards';
+
+export const TermOfUse = () => {
```
Comment (lmazurowski) [inline src/components/TermandPrivacy/TermOfUse.tsx]: Why didn't You use an simplified version? () => (

) ?
And where is FC type for this functional component?

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 10]
```diff
@@ -91,6 +91,7 @@ export const HamburgerMenu: FC<HamburgerMenuType> = ({ isWithHamburger, isWithSe
                 localStorage.removeItem(PS_TOKEN_NAME);
                 client.cache.reset();
                 history.push(`.${APP_ROUTES.LOGIN}`);
+                client.cache.reset();
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx:93]: why we must call reset before and after history push?

File: src/pages/Login/Login.tsx [hunk 11]
```diff
@@ -18,7 +17,7 @@ import { Link } from 'react-router-dom';
 import { PageWrapper } from 'components/PageWrapper/PageWrapper';
 // TODO: Remove after real integration
 import { useUserContext } from 'global/UserContext/UserContext';
-import { signInWithGoogle } from '../../helpers/signInWithGoogle';
+import { APP_ROUTES } from '../../global/AppRouter/routes';
```
Comment (lmazurowski) [inline src/pages/Login/Login.tsx]: You can reduce this path to 'global/AppRouter/routes'

File: src/pages/Login/Login.tsx [hunk 12]
```diff
@@ -169,7 +180,7 @@ export const Login: FunctionComponent = () => {
                 )}
               </StyledCard>
               <RegisterCaption>
-                Don’t have account? <Link to={APP_ROUTES.REGISTER}>Register Now</Link>
+                Don’t have account? <Link to="/register">Register Now</Link>
```
Comment (lmazurowski) [inline src/pages/Login/Login.tsx]: why You replace it with /register. Please - use APP_ROUTES constant.

File: src/pages/Register/Register.tsx [hunk 13]
```diff
@@ -154,7 +162,11 @@ export const Register: FC = () => {
                 <StyledErrorMessage>{apiError}</StyledErrorMessage>
                 <StyledCheckbox name="termsAccepted">
                   <Caption>
-                    I read and agree to <Link to="/terms-and-conditions">Terms & Conditions</Link>
+                    I read and agree to
+                    <LinkStyle to="/termsofuse">Terms of use,</LinkStyle>
```
Comment (lmazurowski) [inline src/pages/Register/Register.tsx]: Please add APP_ROUTES usage here

File: src/pages/Register/Register.tsx [hunk 14]
```diff
@@ -154,7 +162,11 @@ export const Register: FC = () => {
...
+                    <LinkStyle to="/privacyandcookiespolicy">privacy and cookies policy</LinkStyle>
```
Comment (lmazurowski) [inline src/pages/Register/Register.tsx]: Please add APP_ROUTES usage here

File: src/pages/Register/Register.tsx [hunk 15]
```diff
@@ -154,7 +162,11 @@ export const Register: FC = () => {
...
+                    and
+                    <LinkStyleLeft to="/informativeclause">informative clause</LinkStyleLeft>
```
Comment (lmazurowski) [inline src/pages/Register/Register.tsx]: Please add APP_ROUTES usage here

**Review by lmazurowski — APPROVED** [review]

**Pull request merged**